client/src/components/ChatWindow/ChatWindowContainer.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import { webSocketActions } from '../../store/actions';
+import ChatWindow from './ChatWindow';
+
+const mapDispatchToProps = {
+    connectWebSocket: webSocketActions.connectWebSocket
+};
+
+export default connect(null, mapDispatchToProps)(ChatWindow);
```

Comment:
why are you passing null ? is it necessary ?

Reply:
Because I'm only using mapDispatchToProps and not mapStateToProps